src/base/Syntax.ml

```diff
@@ -145,25 +147,35 @@ let address_length = 20
 
 let hash_length = 32
 
+open Integer256
+
+let equal_int128 x y = Int128.compare x y = 0
+let equal_int256 x y = Int256.compare x y = 0
+
 type int_lit =
   | Int32L of int32
   | Int64L of int64
   | Int128L of int128
-  | Int256L of Integer256.int256
+  | Int256L of int256 [@@deriving equal]
```

Comment:
```suggestion
  | Int256L of int256
  [@@deriving equal]
```

Reply:
This suggestion and the next one are just for consistency with the other `@@deriving` directives.

Reply:
I think this is a comment on a commit introduced prior to `make fmt`